File: go-client-sdk/pkg/models/shared/accountcreate.go
package shared

type AccountCreate struct {
	Additional        map[string]interface{} `json:"additional,omitempty"`
	Address           *Address               `json:"address,omitempty"`
	AnnualRevenue     *float64               `json:"annualRevenue,omitempty"`
	Description       *string                `json:"description,omitempty"`
	Industry          *string                `json:"industry,omitempty"`
	Name              string                 `json:"name"`
	NumberOfEmployees *float64               `json:"numberOfEmployees,omitempty"`
	Phone             *string                `json:"phone,omitempty"`
	Website           *string                `json:"website,omitempty"`
}

File: typescript-client-sdk/src/sdk/models/shared/accountcreate.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { Address } from "./address";



export class AccountCreate extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=additional" })
  additional?: Record<string, any>;

  @SpeakeasyMetadata({ data: "json, name=address" })
  address?: Address;

  @SpeakeasyMetadata({ data: "json, name=annualRevenue" })
  annualRevenue?: number;

  @SpeakeasyMetadata({ data: "json, name=description" })
  description?: string;

  @SpeakeasyMetadata({ data: "json, name=industry" })
  industry?: string;

  @SpeakeasyMetadata({ data: "json, name=name" })
  name: string;

  @SpeakeasyMetadata({ data: "json, name=numberOfEmployees" })
  numberOfEmployees?: number;

  @SpeakeasyMetadata({ data: "json, name=phone" })
  phone?: string;

  @SpeakeasyMetadata({ data: "json, name=website" })
  website?: string;
}
